zetaclient/evm_client.go

```diff
@@ -70,6 +73,7 @@ type EVMChainClient struct {
 	*ChainMetrics
 	chain                      common.Chain
 	evmClient                  EVMRPCClient
+	rpcClient                  *ethrpc.EthRPC // a fallback rpc client
```

Comment:
```suggestion
	evmAlternateRpcClient                  *ethrpc.EthRPC // a fallback rpc client
```

Reply:
the [commit](https://github.com/zeta-chain/node/pull/1665/commits/003e851e10e5f76fee146ac76cc4bb564b6c523b)